src/main/scala/scala/sys/process/ProcessWithPid.scala

```diff
@@ -0,0 +1,48 @@
+package scala.sys.process
```

Comment:
This has to be in the `scala.sys.process` package so it has access to `Process.SimpleProcess`, which is package private